src/app/components/utility/modal/modal.component.scss

```diff
@@ -4,3 +4,7 @@
     color: $highlight;
     margin-bottom: 20px;
 }
+
+.modal-card {
```

Comment:
@Suryansh5545 Can you please make this change to `base.scss` Here (https://github.com/Cloud-CV/EvalAI-ngx/blob/master/src/styles/base.scss#L1414).

Also please set it to `auto` so that it can adjust according to it's content for other models.

Reply:
done sir